packages/fast-components-react-base/app/components/react-html-element.tsx

```diff
@@ -15,6 +15,11 @@ export default class ReactHTMLElement extends React.Component<ReactHTMLElementPr
     };
 
     public render(): JSX.Element {
-        return <this.props.tag>{this.props.children}</this.props.tag>;
+        const props: any = Object.assign({}, this.props);
+
+        delete props.tag;
```

Comment:
Not holding up the PR, but what's going on here?

Reply:
this allows managed classes to pass down to the child rendered by the component... Which it was not doing before